### PR TITLE
Add 'unified' theme from Mozilla.com/Bedrock

### PR DESCRIPTION
**Pull request by Osmose**
The unified theme is the current Mozilla.com theme that is intended to be used across most Mozilla properties. Affiliates is going to be using this theme, and I figured now is as good a time as any to add it to django-moz-header so that other django/playdoh based sites can use them easily.

The code for this was taken from Bedrock. The only changes were making image paths in CSS relative and to remove en-US and add mozilla.com to links in the template that needed them.

### CODE REVIEW

File: unified/css/template.css [hunk 1]
```diff
@@ -0,0 +1,711 @@
```
Comment (jsocol) [inline unified/css/template.css:0]: Please be consistent with indents. I believe we've pretty much standardized on 4 spaces for CSS. There's a mix of 2- and 4-space indents in here.

File: unified/footer.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
+<div id="sub-footer">
+  <div id="sub-footer-contents">
+    <h3>{{ _('Let’s be <span>friends!</span>') }}</h3>
```
Comment (jsocol) [inline unified/footer.html]: Does this need `|safe` or did we patch `_` to return `Markup`?

File: unified/css/template.css [hunk 3]
```diff
@@ -0,0 +1,706 @@
...
+    z-index: 99;
+    border: 1px solid #98b2c9;
+    width: 190px;
+    -moz-border-radius: 0 4px 4px 4px;
+    -webkit-border-radius: 0 4px 4px 4px;
+    border-radius: 0 4px 4px 4px;
+    text-shadow: 1px 1px 0 #fff, 0 0 5px #fff;
+    background: #fbfdff;
+    background: rgba(255,255,255,0.9);
+    box-shadow:
```
Comment (potch) [inline unified/css/template.css:216]: we miss 3.6 if we don't have the -moz-box-shadow. I'm fine with that, just making sure you are.

File: unified/css/template.css [hunk 4]
```diff
@@ -0,0 +1,706 @@
...
+}
+
+/* Currently active menu items */
+
+#nav-main ul li.current span,
+#nav-main ul li.current a,
+#nav-main ul li.current a:link,
+#nav-main ul li.current a:visited {
+    border-bottom-width: 1px;
+    -mozborder-radius: 0 0 4px 4px;
```
Comment (potch) [inline unified/css/template.css]: -moz-border-radius

File: unified/css/template.css [hunk 5]
```diff
@@ -0,0 +1,706 @@
...
+    margin: 4px 0 15px 0;
+}
+
+#footer-menu li.last {
+    margin-right: 0;
+}
+
+#footer-menu ul ul li {
+    float: none;
+    padding: 0 0 0.3em 0;
```
Comment (potch) [inline unified/css/template.css]: `.3em` is fine &lt;/totalnitpick&gt;

File: unified/js/nav-main.js [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+        menuitems.mouseover(function(event) {
+            this.focus(); // Sometimes $(this).focus() doesn"t work
+        }).focus(function() {
+            focused_item = $(this);
+        }).each(function(item_idx) {
+            // Enable keyboard navigation
+            $(this).keydown(function(event) {
+                var target;
+                switch (event.keyCode) {
+                case 33: // Page Up
```
Comment (potch) [inline unified/js/nav-main.js]: indent all this one more tab-stop.

File: unified/css/template.css [hunk 7]
```diff
@@ -0,0 +1,706 @@
...
+
+body {
+    text-align: center;
+    font-family: Georgia, serif;
+    margin: 0;
+    min-width: 990px;
+    padding: 0;
+}
+
+#nav-access { display: none; }
```
Comment (tofumatt) [inline unified/css/template.css]: Lay this out like the rest of the CSS; minify will make it small when it needs to be. This is harder to read/expand and it sticks out oddly.

File: unified/css/template.css [hunk 8]
```diff
@@ -0,0 +1,706 @@
...
+    height: 0;
+    clear: both;
+    visibility: hidden;
+}
+
+#wrapper {
+    min-width: 980px;
+}
+
+.hide { display: none; }
```
Comment (tofumatt) [inline unified/css/template.css]: Newline/indent this too.

File: unified/css/template.css [hunk 9]
```diff
@@ -0,0 +1,706 @@
...
+#nav-main ul li.current span,
+#nav-main ul li.current a,
+#nav-main ul li.current a:link,
+#nav-main ul li.current a:visited {
+    border-bottom-width: 1px;
+    -mozborder-radius: 0 0 4px 4px;
+    -webkit-border-radius: 0 0 4px 4px;
+    border-radius: 0 0 4px 4px;
+    background: #fbfdff;
+    background: -moz-linear-gradient(top, rgba(255,255,255,0.4) 0%, rgba(255,255,255,0.8) 100%);
```
Comment (tofumatt) [inline unified/css/template.css]: A bit nit-picky, but you could split this up so it was under 72 chars wide.

File: unified/css/template.css [hunk 10]
```diff
@@ -0,0 +1,706 @@
...
+    text-shadow: 1px 1px 2px rgba(0,0,0,0.5), 0 0 5px rgba(51,85,155,1), 0 0 20px rgba(255,255,255,0.6);
+    -moz-transition: text-shadow 0.1s ease-out;
+    -webkit-transition: text-shadow 0.1s ease-out;
+    -o-transition: text-shadow 0.1s ease-out;
+    transition: text-shadow 0.1s ease-out;
+}
+
+#sub-footer ul li#footer-twitter  a { background-position: 50% -115px; }
+#sub-footer ul li#footer-facebook a { background-position: 50% -215px; }
+#sub-footer ul li#footer-connect  a { background-position: 50% -315px; }
```
Comment (tofumatt) [inline unified/css/template.css]: Lay these out like other CSS (newline after curly brace; indentation).

File: unified/css/template.css [hunk 11]
```diff
@@ -0,0 +1,706 @@
...
+    margin: 10px 0 4px 0;
+    display: block;
+}
+
+#lang_form select {
+    width: 140px;
+    font-family: sans-serif;
+}
+
+#lang_form div { display: inline; }
```
Comment (tofumatt) [inline unified/css/template.css]: Re: style -- this too.

File: unified/css/template.css [hunk 12]
```diff
@@ -0,0 +1,706 @@
...
+    zoom: 1;
+}
+
+#content {
+    margin: 0 20px;
+}
+
+/* Larger text for handheld devices */
+@media handheld {
+  #main-content, #main-feature { font-size: 130%; }
```
Comment (tofumatt) [inline unified/css/template.css]: 4 spaces for indents, style here.

File: unified/css/template.css [hunk 13]
```diff
@@ -0,0 +1,706 @@
...
+}
\ No newline at end of file
```
Comment (tofumatt) [inline unified/css/template.css:776]: Newlines at EOF are cool.

File: unified/footer.html [hunk 14]
```diff
@@ -0,0 +1,121 @@
+<div id="sub-footer">
+  <div id="sub-footer-contents">
+    {% block sub_footer %}
+      <h3>{{ _('Let’s be <span>friends!</span>') }}</h3>
+      <ul>
+        <li id="footer-twitter"><a href="http://twitter.com/firefox">{{ _('Twitter') }}</a></li>
+        <li id="footer-facebook"><a href="http://Facebook.com/Firefox">{{ _('Facebook') }}</a></li>
+        <li id="footer-connect"><a href="http://www.mozilla.com/firefox/connect/">{{ _('More Ways to Connect') }}</a></li>
```
Comment (tofumatt) [inline unified/footer.html]: These will fit in 72 chars if you throw some linebreaks between the tags/translation string.

File: unified/footer.html [hunk 15]
```diff
@@ -0,0 +1,121 @@
...
+      </ul>
+    </div>
+    {# end footer menu #}
+
+    <div id="copyright">
+      <p id="footer-links">
+        <a href="http://www.mozilla.com/privacy-policy.html">{{ _('Privacy Policy') }}</a> &nbsp;|&nbsp;
+        <a href="http://www.mozilla.com/about/legal.html">{{ _('Legal Notices') }}</a> &nbsp;|&nbsp;
+        <a href="http://www.mozilla.com/legal/fraud-report/index.html">{{ _('Report Trademark Abuse') }}</a><br/>
+      </p>
```
Comment (tofumatt) [inline unified/footer.html]: This should be a `<ul>` with some margins. `<p>` tag is for prose.

**Comment by potch** [comment]
Looking good! Thanks for all the style tweaks.

**Pull request merged**